### PR TITLE
Slider update on input

### PR DESCRIPTION
**Pull request by lahmann**
When digits are entered into the form fields of special facet dateRange the slider does not get updated. This js adds updating of slider for better feedback for the user.

### CODE REVIEW

**Comment by demiankatz** [comment]
Looks good to me. Just one question: should we include better NaN handling? Right now, if I put non-numeric text into the input fields, it translates into "NaN." Would it be better to treat NaN the same as 0?

**Comment by lahmann** [comment]
Makes definitely sense - good catch! I updated the pull request accordingly.

File: themes/bootstrap3/templates/Recommend/SideFacets.phtml [hunk 1]
```diff
@@ -123,6 +123,17 @@ $(document).ready(function() {
   .on('slide', fillTexts)
   .data('slider');
 });
+
+$('#{$this->escapeHtmlAttr($title)}from, #{$this->escapeHtmlAttr($title)}to').change(function () {
+  $('#{$this->escapeHtmlAttr($title)}dateSlider').slider(
+    'setValue',
+    [
+      isNaN(Number($('#{$this->escapeHtmlAttr($title)}from').val())) || Number($('#{$this->escapeHtmlAttr($title)}from').val()) <= 0 ? {$min} : Number($('#{$this->escapeHtmlAttr($title)}from').val()),
+      isNaN(Number($('#{$this->escapeHtmlAttr($title)}to').val())) || Number($('#{$this->escapeHtmlAttr($title)}to').val()) <= 0 ? {$max} : Number($('#{$this->escapeHtmlAttr($title)}to').val())
```
Comment (crhallberg) [inline themes/bootstrap3/templates/Recommend/SideFacets.phtml]: I'd recommend breaking this into smaller steps for readability. Also, saving `Number($('#{$this->escapeHtmlAttr($title)}from').val())` to a variable will prevent from repeating a jQuery search and conversion twice.

**Comment by lahmann** [comment]
@crhallberg, I agree and updated the pull request - imo readability also improved by usage of variables for from/toValues. What do you think?

**Comment by demiankatz** [comment]
Looks pretty good, but any reason not to also move the Number() part up into the variable assignment? Right now we don't appear to use the raw values anywhere, and we're applying Number() three separate times for each variable.

**Comment by demiankatz** [comment]
I refactored the Number() calls as suggested and merged this. Thanks again!

**Pull request merged**